index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+ <meta charset="UTF-8">
+ <meta http-equiv="X-UA-Compatible" content="IE=edge">
+ <meta name="viewport" content="width=device-width, initial-scale=1.0">
+ <link rel="stylesheet" href="styles.css">
+ <link rel="preconnect" href="https://fonts.gstatic.com">
+<link href="https://fonts.googleapis.com/css2?family=Poppins:ital@1&display=swap" rel="stylesheet">
+<title>Portfolio</title>
+</head>
+<body>
+ <header> <nav id="menu-top">
+  <ul class="topnav">
+    <li class="topnav-logo"><a href="#">My Logo</a></li>
+    <li class="topnav-right"><a href="#">Home</a></li>
+    <li class="topnav-right"><a href="#">About</a></li>
+    <li class="topnav-right"><a href="#">Portfolio</a></li>
+    <li class="dropdownIcon"><a href="#">&#9776;</a></li>
+  </ul>
+</nav>  </header>
+ 
+ <div class="flex-container">
+   <div class="name">
+     <h1>I’m Lilian 
+    Glad to see you!</h1>
+     </div>
+  <div class="description"><p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me</p></div>
```

Comment:
- [ ] I'm sure the design could improve if you take some minutes to find a font more like the original design on the Figma template. I kindly invite you to take some time to choose a font more about this. 🐦